ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__ppc64le-nightly.yaml

```diff
@@ -88,6 +88,21 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: ipi-powervs
+- as: powervs-ipi-e2e-test-qe-destructive
+  cluster: build01
+  cron: 0 22 * * 2
+  steps:
+    cluster_profile: powervs-1
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:ppc64le-latest
+    env:
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ppc64le and @baremetal-upi and @destructive and @network-ovnkubernetes
+        and not @fips'
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe
```

Comment:
For destructive tests, we are supposed to use "chain: openshift-e2e-test-qe-destructive"